_targets/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
It's better to have a single `.gitignore`, otherwise it can get confusing as to why certain files are ignored. If you need to make this directory persist maybe you can add a README with just a line explaining what it is or something. 

But do we need to commit this directory?

Reply:
this is automatically generated by `_targets` and my understanding is that it is best to leave it as is to avoid complications

https://github.com/ropensci/targets/issues/685

I see alot of R packages and workflows that auto-generate more than one gitignore in a sub-directories (`{usethis}`.`{targets}`, `{shiny}`) so maybe it will just be easier to provide a better description in the readme as you suggest

Reply:
Ah okay, it's not ideal but if it's auto-generated then fine